config/settings.py

```diff
@@ -67,9 +70,16 @@
     AWS_MEDIA_LOCATION=(str, 'media'),
 
     # Http protocol settings
-    HTTP_PROTOCOL=(str, 'http')
+    HTTP_PROTOCOL=(str, 'http'),
+
+    # Sentry
+    SENTRY_DSN=(str, None),
+    SENTRY_SAMPLE_RATE=(float, 0.2),
 )
 
+IFRC_GATES_ENVIRONMENT=env('IFRC_GATES_ENVIRONMENT')
+APP_TYPE=env('APP_TYPE'),
```

Comment:
```suggestion
APP_TYPE=env('APP_TYPE')
```

Reply:
instead of IFRC_GATES_*, can we name it like RCCE_CS (for RCCE COLLECTIVE SERVICE) wherever possible ?

Reply:
instead of IFRC_GATES_*, can we name it like RCCE_CS (for RCCE COLLECTIVE SERVICE) wherever possible ?